roles/kubernetes/master/defaults/main.yml

```diff
@@ -88,3 +88,8 @@ kubeadm_token_ttl: 0
 
 ## Variable for influencing kube-scheduler behaviour
 volume_cross_zone_attachment: false
+
+## Encrypting Secret Data at Rest
+encrypt_secret_data: false
+encrypt_token: "{{ lookup('password', 'credentials/encrypt_token length=32 chars=ascii_letters,digits') }}"
```

Comment:
This was recently changed so that it went inside inventory dir:
```encrypt_token: "{{ lookup('password', inventory_dir + 'credentials/encrypt_token length=32 chars=ascii_letters,digits') }}" 
```

Reply:
I dont get this comment?